src/main/resources/META-INF/rust-core.xml

```diff
@@ -1223,6 +1223,9 @@
         <registryKey key="org.rust.code.vision.author"
                      defaultValue="false"
                      description="Enable version control author Code Vision hints for Rust" />
+        <registryKey key="org.rust.code.vision.usage"
```

Comment:
Let's move the registry key to `platform-rust-core.xml`

Reply:
Why?

Reply:
To make it available only with 223, since the usages hints are not available with 222 anyway